keps/sig-network/1880-multiple-service-cidrs/kep.yaml

```diff
@@ -20,27 +20,28 @@ see-also:
 replaces:
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: alpha
+stage: beta
 
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.29"
+latest-milestone: "v1.31"
 
 # The milestone at which this feature was, or is targeted to be, at each stage.
 milestone:
   alpha: "v1.27"
-  beta: "v1.30"
-  stable: "v1.32"
+  beta: "v1.31"
+  stable: "v1.33"
```

Comment:
Let's leave the stable stage not filled, until you actually get there. It confuses tooling and folks looking at annual reports which scrape these information :wink:

Reply:
yep